chroma-agent/Target

```diff
@@ -103,7 +103,7 @@ Target_stop() {
 }
 
 Target_status() {
-    dbg "Target_status()"
+    # dbg "Target_status()"
```

Comment:
Can we just remove this instead of commenting

Reply:
will do next repush